### ISSUE TITLE
Release 1.0.1

### ISSUE THREAD

**Issue by mmertsock** [CareEvolution/MyDataHelpsKit-iOS#10]
Prerequisite: iOS docs fully live at https://developer.rkstudio.careevolution.com. Afterward, update this repo's readme with a more prominent link to the documentation, etc., and publish this release.

## Checklist

- [x] Create a new branch `release/1.0.1` off of `main`
- [x] In Xcode, select the MyDataHelpsKit target, and set the correct version and build numbers in Project Settings
- [x] Compile MyDataHelpsKit, which regenerates SDKVersion.swift
- [x] Commit the changes to project.pbxproj and SDKVersion.swift files
- [x] Create a pull request with your changes
- [ ] Regenerate documentation and create a separate PR for publishing the docs
- [x] After merging the pull request, create a release tag in GitHub. Swift Package Manager requires the tag to be in format X.Y.Z (full semantic version); include the patch number even if it is `.0`
